components/encoder/single/single_encoder_test.go

```diff
@@ -112,8 +112,7 @@ func TestEncoder(t *testing.T) {
 		})
 	})
 
-	// this test ensures that digital interrupts are ignored if AttachDirectionalAwareness
```

Comment:
Do we have a test that tests the new logic after a reconfiguration? 
The following case I want addressed:
- Added a motor with directional awareness.
- Tick back and forth.
- Removed the directional awareness
- Tick forward only.
- Added back the motor with directional awareness.
- Tick back and forth.